### PR TITLE
Fixes #17521: A repaired in system policies should not be seen as a problem for warning display in nodes list

### PR DESCRIPTION
**Pull request by VinceMacBuche**
https://issues.rudder.io/issues/17521

### CODE REVIEW

File: webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js [hunk 1]
```diff
@@ -1537,6 +1538,14 @@ function computeComplianceOK (complianceArray) {
     return { percent: 0, number: 0};
   }
 }
+
+function reportsSum (complianceArray) {
+  if (Array.isArray(complianceArray)) {
+    return complianceArray.reduce(function(total, value) { return total + reduce} )
```
Comment (fanf) [inline webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js:1544]: what happen for reduce when array size is 1? Does it correcty returns the value, or does it fails ?

Comment (fanf) [reply webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js:1544]: should be `total+value`
(and it's ok for size == 1, just checked)

**Comment by VinceMacBuche** [comment]
PR updated with a new commit

**Comment by fanf** [comment]
OK, squash merging this PR

**Comment by fanf** [comment]
OK, merging this PR

**Pull request merged**